Toolset/home.livecodescript

```diff
@@ -723,12 +672,359 @@ private command revInternal__SetupSplash
 	palette stack "revSplash"
 end revInternal__SetupSplash
 
--- MW-2011-03-10: Ensure certain preference settings reset to what 'new' defaults should be
---   (This was easier than attempting to edit revpreferences.rev which is very very slippery!)
 command revInternal__ResetPreferences
-   set the cREVToolsPaintToolsVisible of stack "revPreferences" to true
-   set the cSEHidePalettes of stack "revPreferences" to false
-   set the cSEHideMessageBox of stack "revPreferences" to false
+   -- clear the preferences stack
+   set the customKeys of stack "revPreferences" to empty
+   set the customPropertySets of stack "revPreferences" to empty
+   
+   local tPropsArray
+   -- default custom property set
+   put "true" into tPropsArray["cREVToolsPaintToolsVisible"]
+   put "LightBlue" into tPropsArray["LightBlue"]
+   put "false" into tPropsArray["cDestroyStack"]
+   put "false" into tPropsArray["cRevPopups"]
+   put "417,113" into tPropsArray["cREVMsgTabRectGlobalProperties"]
+   put "155" into tPropsArray["cREVPlayerWidth"]
+   put "23" into tPropsArray["cButtonHeight"]
+   put "true" into tPropsArray["cShowProfileEditor"]
+   put "200" into tPropsArray["cScrollbarwidth"]
+   put "false" into tPropsArray["cREVMsgShowUIMessages"]
+   put "false" into tPropsArray["cSEHidePalettes"]
+   put "120" into tPropsArray["cREVImageWidth"]
+   put "500" into tPropsArray["tooltip delay"]
+   put "" into tPropsArray["cUserOrganization"]
+   put "417,72" into tPropsArray["cREVMsgTabRectSingle Line"]
+   put "false" into tPropsArray["cREVMsgShowUIBackScripts"]
+   put "539,432" into tPropsArray["cREVGeoTopLeft"]
+   put "70378" into tPropsArray["cREVDBCode"]
+   put "" into tPropsArray["cColumnVisibility"]
+   put "276" into tPropsArray["cREVScriptHeight"]
+   put "2" into tPropsArray["cGridSize"]
+   put "483,214,1015,476" into tPropsArray["cVariableWatcherRect"]
+   put "false" into tPropsArray["cShowrevProps"]
+   put "Pixels" into tPropsArray["cREVRulerMeasurement"]
+   put "GreenYellow" into tPropsArray["GreenYellow"]
+   put "100" into tPropsArray["cMacOSScrollbarWidth"]
+   put "1,2,3,4,5,7" into tPropsArray["cControlTabs"]
+   put "" into tPropsArray["cREVCustomWidth"]
+   put "500" into tPropsArray["cREVToolTipDelay"]
+   put "250" into tPropsArray["cTabButtonwidth"]
+   put "true" into tPropsArray["cGoAlign"]
+   put "1:80" into tPropsArray["http proxy"]
+   put "2.6.1" into tPropsArray["cVersionNo"]
```

Comment:
Quite possibly a legacy from the days when the engine version <> the IDE version